### ISSUE TITLE
请问一个使用PGnet训练弯曲文本印章的问题

### ISSUE THREAD

**Issue by austinleeee** [PaddlePaddle/PaddleOCR#9096]
在训练轮次靠前时，其评估指标有明显提升，轮次约靠后则训练效果越差，但是loss损失依然是又在减少，这是怎么个问题？
数据集使用的：[电子印章数据集]
预训练模型：加载的是预训练模型wget -P ./pretrain_models/ https://paddleocr.bj.bcebos.com/dygraph_v2.0/pgnet/train_step1.tar

第一轮结果:
![image](https://user-images.githubusercontent.com/116559785/218991740-a79aff4a-a9cf-491b-b9f0-dcfba339e0b0.png)
第二轮：
![image](https://user-images.githubusercontent.com/116559785/218991877-dce582f3-8d72-4156-8ff6-59ebd5ddcf86.png)
第三轮
![image](https://user-images.githubusercontent.com/116559785/218991937-f77ff4a4-62bd-43a6-983a-57fcb96c51bf.png)
第四轮：
![image](https://user-images.githubusercontent.com/116559785/218992010-da11357d-89ba-4b53-802e-7279ad0e5ed0.png)
第五轮：
![image](https://user-images.githubusercontent.com/116559785/218992082-15cac65d-0c53-4c5a-bc2c-27b224663c87.png)

# 配置文件：
Global:
  # 使用预训练模型
  # 标签14points
  use_gpu: true
  epoch_num: 5
  log_smooth_window: 20
  print_batch_step: 10
  #
  save_model_dir: ./models/pgnet_model_pre8_1/
  save_epoch_step: 20
  eval_batch_step:
  - 0
  - 1000
  cal_metric_during_train: false
  #
  pretrained_model: ./pretrain_models/pgnet_pretrained/best_accuracy
  # best_accuracy or latest
  checkpoints: ./models/pgnet_model_pre8_1/latest
  save_inference_dir: null
  use_visualdl: false
  infer_img: eval_data/seal3.png
  valid_set: totaltext
  #
  infer_visual_type: CN
  #
  save_res_path: ./output/pgnet_pre8_3/predicts_pgnet.txt
  character_dict_path: ppocr/utils/ppocr_keys_v1.txt
  #
  character_type: CN
  max_text_length: 20
  max_text_nums: 4
  # 表示文本中心线上最多采样多少次（字符）
  tcl_len: 64
  load_static_weights: false
Architecture:
  model_type: e2e
  algorithm: PGNet
  Transform: null
  Backbone:
    name: ResNet
    layers: 50
  Neck:
    name: PGFPN
  Head:
    name: PGHead
    character_dict_path: ./ppocr/utils/ppocr_keys_v1.txt
Loss:
  name: PGLoss
  tcl_bs: 64
  max_text_length: 20
  max_text_nums: 4
  pad_num: 6623
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 50
  regularizer:
    name: L2
    factor: 5.0e-05
PostProcess:
  name: PGPostProcess
  score_thresh: 0.5
  mode: fast
Metric:
  name: E2EMetric
  mode: A
  gt_mat_dir: ./train_data/seal_dataset/test_label.txt
  character_dict_path: ppocr/utils/ppocr_keys_v1.txt
  main_indicator: f_score_e2e
Train:
  dataset:
    name: PGDataSet
    data_dir: ./train_data/seal_dataset/train
    label_file_list: [./train_data/seal_dataset/train/train.txt]
    ratio_list: [1.0]
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - E2ELabelEncodeTrain: null
    - PGProcessTrain:
        batch_size: 8
        use_resize: True
        use_random_crop: False
        min_crop_size: 24
        min_text_size: 4
        max_text_size: 512
        point_gather_mode: align
    - KeepKeys:
        keep_keys:
        - images
        - tcl_maps
        - tcl_label_maps
        - border_maps
        - direction_maps
        - training_masks
        - label_list
        - pos_list
        - pos_mask
  loader:
    shuffle: true
    drop_last: true
    batch_size_per_card: 8
    num_workers: 16
Eval:
  dataset:
    name: PGDataSet
    data_dir: ./train_data/seal_dataset/test
    label_file_list: [./train_data/seal_dataset/test/test.txt]

    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - E2ELabelEncodeTest:
        character_dict_path: ppocr/utils/ppocr_keys_v1.txt
        max_side_len: 768
    - NormalizeImage:
        scale: 1./255.
        mean: [ 0.485, 0.456, 0.406 ]
        std: [ 0.229, 0.224, 0.225 ]
        order: 'hwc'

    - DetResizeForTest:
        image_shape:
        - 512
        - 512
        padding: true
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - shape
        - polys
        - texts
        - ignore_tags
        - img_id
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 1
    num_workers: 2

**Comment by LDOUBLEV** [comment]
把训练数据集的一部分拿出来评测看下，难道是过拟合？不过评估指标还那么低，不像是过拟合了

**Comment by austinleeee** [comment]
> 把训练数据集的一部分拿出来评测看下，难道是过拟合？不过评估指标还那么低，不像是过拟合了

请问一般训练一个PGnet的中文模型需要多少的数据量

**Comment by wangxinzhe0617** [comment]
我训练也碰到了这个问题，我那个更惨直接召回精度都是0.。。。。。。。。楼主这个问题解决了不。那个ctc_loss一直降不下去，大概在38左右。

**Comment by austinleeee** [comment]
> 我训练也碰到了这个问题，我那个更惨直接召回精度都是0.。。。。。。。。楼主这个问题解决了不。那个ctc_loss一直降不下去，大概在38左右。

现有端到端针算法对中文的弯曲文本效果不好，我没有在尝试端到端的算法了
